models/base_sources/polygon_base_sources copy.yml

```diff
@@ -0,0 +1,37 @@
+version: 2
```

Comment:
can you fix the filename? it's `polygon_base_sources copy`

Reply:
Fixed.